src/test/scala/nl.knaw.dans.easy.multideposit/UnitSpec.scala

```diff
@@ -22,7 +22,7 @@ import nl.knaw.dans.easy.multideposit.model._
 import org.joda.time.DateTime
 import org.scalatest._
 
-abstract class UnitSpec extends FlatSpec with Matchers with OptionValues with Inside with OneInstancePerTest {
+abstract class UnitSpec extends FlatSpec with Matchers with OptionValues with Inside {
```

Comment:
`OneInstancePerTest` gives worse performance and results in errors in the new `BlackBoxSpec`